schema/artist/index.js

```diff
@@ -193,6 +195,21 @@ export const ArtistType = new GraphQLObjectType({
           )
         },
       },
+      auctionResults: {
+        type: auctionResultConnection,
+        args: pageable({}),
+        resolve: ({ _id }, options, _request, { rootValue: { auctionLotLoader } }) => {
+          // Convert `after` cursors to page params
+          const { page, size, offset } = parseRelayOptions(options)
+          const diffusionArgs = { page, size, artist_id: _id }
```

Comment:
Diffusion notably only supports `page/size` for pagination (and not `size/offset`, which Gravity does (in fact it supports both)).

Reply:
Interesting. I thought that we were also using `page/size` for Gravity pagination in various places, or am I just remembering the non-connection fields and have we been using `size/offset` for connections that hit Gravity?

#lazy-comment

Reply:
Our Gravity V2 API supports `size` & `offset` or alternatively `cursor`. I'd like to support cursor pagination in Diffusion. I could add support for `size/offset` as it is just a derivation of `page/size`.

Reply:
Ahh, I see, Gravity _V2_. Thanks for the info 👍

Reply:
V1 supports `page/size` and `size/offset`, I'm pretty sure I've seen examples of both around here (will look). I suppose that's totally fine since the upstream API won't be changing/breaking that pagination, just prob a bit confusing.